Test/JsonDiffTest.cs

```diff
@@ -0,0 +1,34 @@
+using c_sharp_json_diff;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Test
+{
+    public class JsonDiffTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void WhenJsonDiffOnTwoStringsThenCorrectValueReturned()
+        {
+            string left = @"{
+                              'number': 10,
+                              'sommething': 20
+                             }";
+            string right = @"{
+                              'number': 5,
+                              'sommething': 10
+                             }";
+            JObject diffObject = JsonDiff.NumericDiff(left, right);
+            JArray array = (JArray)diffObject.Value<JToken>("number");
```

Comment:
Before converting to a ``JArray``, can you do an ``Assert`` to confirm that the value is indeed a ``JArray``? This way, test failure will be easier to debug if for some reason a code change breaks the type of this field and it's no longer a ``JArray``

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix